Dockerfile.keycloak

```diff
@@ -15,7 +15,7 @@ COPY cli /opt/jboss/tools/cli
 RUN export KEYCLOAK_HTTP_PORT=8080 && \
       cd  /opt/jboss/keycloak/modules/system/layers/base/com/oracle/jdbc/main/ && \
       mkdir driver && \
-      curl https://repo1.maven.org/maven2/com/oracle/ojdbc/ojdbc8/19.3.0.0/ojdbc8-19.3.0.0.jar -o driver/ojdbc.jar
+      curl https://repo1.maven.org/maven2/com/oracle/database/jdbc/ojdbc8/21.4.0.0.1/ojdbc8-21.4.0.0.1.jar -o driver/ojdbc.jar
```

Comment:
I suggest to put the driver in a generic folder and use it here and other Dockerfile. In this way we avoid duplicating this declaration

Reply:
I'm a bit worried about repo size over time. These jar files are about 5 MB. If we are going to update them frequently, shouldn't be better to retrieve them using curl in both the images rather than committing them to git? I'm just thinking, not sure about pros and cons.

Reply:
Of course it's better, I just proposed to reuse the already uploaded file for a faster change but you can do the opposite if you prefer

Reply:
Added curl commands to the other Dockerfiles and removed binaries